README.md

```diff
@@ -1,38 +1,81 @@
+
+Make sure you have the `python3.7` binary in your PATH. You can also use the `PYTHON` environment variable to override the location of the
+`python` binary. *e.g.:*
+```bash
+$ PYTHON=/usr/bin/python3.7 make
+```
+
+## Database schema
+
+Before running the alembic migrations make sure you  have created a database called `newdle` (or adjust the config file). Having done that, run `flask db upgrade` to upgrade the schema.
+
+**:warning: Schema changes :warning:**
+
+At the initial stages of the development it might be pretty common to change the DB schema which also incurs modifications to the SQLAlchemy models. To facilitate the process of keeping actual schema and models in sync we provide `make newdb` command. The purpose of it is to update the initial alembic revision (according to the models declared) and recreate your local database. This is a destructive operation! This command will be removed as soon as there are more alembic revisions.
+
+
+## Running the development server
+
+To run the dev servers, use `make flask-server` and `make react-server` (in separate terminals). You can use the `FLASK_HOST`, `FLASK_PORT` and `REACT_PORT` environment variables to override where the dev servers will listen (make sure to set it for both dev servers, since the react server needs to know where the Flask app is running).
+
+Once everything is running, you can access the webapp on `http://127.0.0.1:3000` if you did not change any of the ports.
 
-To run the dev servers, use `make flask-server` and `make react-server` (in separate terminals).
-You can use the `FLASK_HOST`, `FLASK_PORT` and `REACT_PORT` environment variables to override
-where the dev servers will listen (make sure to set it for both dev servers, since the react
-server needs to know where the Flask app is running).
+Use the `BROWSER` environment variable if you want to prevent new browser windows being opened every time you run `make react-server`.
 
-Once everything is running, you can access the webapp on `http://127.0.0.1:3000` if you did not
-change any of the ports.
+```bash
+BROWSER=none make react-server
+```
 
-Code formatting tools are automatically executed when you commit, but you can run `make format`
-to run them manually.
+## Other available `make` targets
 
-The unit tests can be run using `make test`. Linting (with pycodestyle) can be done using `make lint`.
+We provide a couple of additional `make` targets that should streamline the development process:
 
-After updating the database structure, run `make newdb` to update the initial alembic revision
-and recreate your local database. This is a destructive operation! This command will be removed
-as soon as there are more alembic revisions.
+ - `make clean` - removes all generated files
+ - `make distclean` - runs `clean` target first and removes config files afterwards;
```

Comment:
I'd add semicolon on either all the bullet points or none.

Reply:
+1 on "none"

Reply:
Oh, good catch. I don't know how I ended up with only one semicolon :P I will remove it ofc.